server/view/view.controller.ts

```diff
@@ -118,6 +118,22 @@ export class ViewController {
             .render(req, res, "/404-page", Object.assign(parsedUrl.query));
     }
 
+    @IsPublic()
```

Comment:
question: should this be public? The user needs to be logged in so maybe we should enforce that as well.